clients/api-proxy/rne/index.ts

```diff
@@ -57,6 +58,20 @@ const mapToDomainObject = ({
   identite,
   dirigeants,
 }: IRNEProxyResponse) => {
+  const newDirigeants = dirigeants.map((dirigeant) => {
+    if (!isPersonneMorale(dirigeant)) {
+      const prenom = dirigeant.prenom || '';
+
+      return {
+        ...dirigeant,
+        prenom: prenom.split(',')[0],
+        prenoms: prenom,
```

Comment:
Why keep both ?

Reply:
For consistency with Api Recherche Enreprise.
But maybe we could not use it anymore ?